region/maridia/inner-pink/East Cactus Alley.json

```diff
@@ -747,6 +781,31 @@
         "Only unmorphing on the first possible frame will work; second frame will bonk the platform."
       ]
     },
+    {
+      "link": [2, 2],
+      "name": "X-Mode Shinecharge, R-Mode Pause Abuse Spark Interrupt",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        "Gravity",
+        "h_CrystalFlashForReserveEnergy",
+        "h_usePowerBomb",
+        "canOffScreenMovement",
+        "canBeVeryPatient",
+        "canXMode",
+        "h_XModeSpikeHit",
+        "h_shinechargeMaxRunway",
+        "canNeutralDamageBoost",
+        {"autoReserveTrigger": {}},
+        "canRModePauseAbuseSparkInterrupt"
+      ],
+      "resetsObstacles": ["R-Mode"],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Crystal Flash at the door, and enter X-Mode on the spikes and shinecharge.",
+        "Pause abuse on the spikes: land on the spike three tiles from the ledge and face left to neutral damage boost back to safety before the pause."
```

Comment:
For me, damage boosting (neutral or horizontal) doesn't seem to work to get onto the ledge. So I have to take a spike hit after the reserve trigger.